src/ralph/admin/templates/admin/change_list.html

```diff
@@ -89,24 +94,24 @@
 {#             </div> #}
 {#           {% endif %} #}
 {#       </div> #}
-
       <div class="row content with-tabs">
-        {% block filters %}
-          {% if cl.has_filters %}
-            <div class="small-2 columns">
-              {% admin_actions %}
-              <div class="panel">
-                <h4>Filters</h4>
-                {% for spec in cl.filter_specs %}
-                  {% admin_list_filter cl spec %}
-                {% endfor %}
-              </div>
+        {% if not bulk_edit %}
```

Comment:
We should watch template files. This gonna soon look like ified and tagged soup (if already isn't, imho).